x-pack/plugins/monitoring/public/components/elasticsearch/nodes/cells.js

```diff
@@ -27,7 +27,7 @@ const TRENDING_UP = i18n.translate('xpack.monitoring.elasticsearch.node.cells.tr
 });
 
 function OfflineCell() {
-  return <div className="monTableCell__number monTableCell__offline">N/A</div>;
+  return <div className="monTableCell__offline">N/A</div>;
```

Comment:
This is causing a snapshot test to fail.

Reply:
Pushed a fix